pkg/transformers/transformers.go

```diff
@@ -0,0 +1,79 @@
+package transformers
+
+import (
+	"strings"
+)
+
+func Camel(name string) string {
+	var parts []string
+	for i, part := range strings.Split(name, "_") {
+		firstChar := part[0:1]
```

Comment:
How does this behave for a secret name like `TEST_` or simply `_`? Both of these names are apparently allowed in our system 😅 

It might be good to add these to the test cases for all transformers

Reply:
Added tests for `TEST_` and all transformer results are now consistent with server.